.travis.yml

```diff
@@ -19,6 +19,11 @@ env:
 before_script:
   - phpenv config-rm xdebug.ini || echo "xdebug not available"
   - sh -e ./.travis.scripts/compile.sh
+  # Add an IPv6 config - see the corresponding Travis issue
+  # https://github.com/travis-ci/travis-ci/issues/8361
+  - if [ "${TRAVIS_OS_NAME}" == "linux" ]; then
+      sudo sh -c 'echo 0 > /proc/sys/net/ipv6/conf/all/disable_ipv6';
```

Comment:
The moment you add a sudo, Travis switches to a ``sudo: true`` version of their environment.

Reply:
Yes, this was the workaround that I found. Other people reported that it did not work to edit the hosts file with sudo, so I haven't tried that yet.